gui-libs/libhandy/libhandy-1.0.2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gnome.org meson xdg vala virtualx
+
+DESCRIPTION="Library with GTK widgets for mobile phones"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libhandy/"
+
+LICENSE="LGPL-2.1+"
+SLOT="1.0/0"
```

Comment:
This should be SLOT="1", because they (as I understand it) guarantee ABI stability now within the parallel-installable package, and the package is called libhandy-1, not libhandy-1.0